libcxx/test/std/algorithms/alg.nonmodifying/alg.fold/left_folds.pass.cpp

```diff
@@ -50,6 +51,8 @@
 #endif
 
 using std::ranges::fold_left;
+using std::ranges::fold_left_first;
+using std::ranges::fold_left_first_with_iter;
```

Comment:
We prefer not to do this.

Also we prefer to have one test per function overload.
For tests of a set of properties we sometimes use one test like for the nodiscard test.

Reply:
Now I've split this file into four algorithms, each for one algorithm.

Reply:
I'm not sure what should do next... Should we cover different overloads in different function call entries, or in different test files?